File: app/models/booking.rb
class Booking < ApplicationRecord
  belongs_to :house

  validates :day, presence: true, uniqueness: true
  validates :number_of_guests, presence: true, inclusion: { in: 1..8, message: 'the number of guests should be between 1 and 8.' }

  scope :ordered, -> { order(:day) }

  def self.booked_for?(date)
    where(day: date).present?
  end
end

File: src/Views/bookings/_booking.php
<div class='booking'>
  <p><strong>Day:</strong> <?= htmlspecialchars($booking->day) ?></p>
  <p><strong>House:</strong> <?= htmlspecialchars($booking->house()->name ?? 'Unknown') ?></p>
  <?php if ($booking->number_of_guests): ?>
    <p><strong>Guests:</strong> <?= htmlspecialchars($booking->number_of_guests) ?></p>
  <?php endif; ?>
</div>